policies/tiles.md

```diff
@@ -21,7 +19,7 @@ But because OpenStreetMap data is free, many other organisations provide map til
 * Clearly display [license](https://wiki.openstreetmap.org/wiki/License) attribution.
 * Do not actively or passively encourage copyright infringement.
 * Calls to **/cgi-bin/export** may only be triggered by direct end-user action. (For example: "click here to export".) The export call is an expensive (CPU+RAM) function to run and will frequently reject when server is under high load.
-* Highly Recommended: Do not hardcode any URL at tile.openstreetmap.org into an app.
+* Recommended: To prevent needing to create new version of an app in case it gets blocked due to imposing an excessive load, do not hardcode any URL at tile.openstreetmap.org into an app.
```

Comment:
I think it would be clearer to say:

* Recommended: Do not hardcode any URL at tile.openstreetmap.org as doing so will limit your ability to react quickly if the service is disrupted or blocked.